src/FAQ.mdx

```diff
@@ -0,0 +1,46 @@
+
+## Who is the design system for?
+
+The Blue Design System delivers a single source of truth for designers and developers to ease the creation of high quality user experiences.
+
+The Blue Design System is built with the following personas in mind:
+
+- The UX Designer, who wants to provide user experiences that are high quality, accessible, and consistent
+- The UI Developer, who wants their code to be clean and consistent and conform to accessibility standards
+- The Product Owner, who wants their product teams to focus on the hard stuff not the easy stuff so product development moves faster and delivers consistent and reliable product experiences every time
+- The Product Stakeholders, who want to be assured of the accessibility, consistency, and quality of the user experience
+
+## How do I get the latest Blue Design System library?
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+## How can I help with the Blue Design System?
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+## What language are the components written in?
+
+## SOS!
+
+I see a bug, how do I know if it’s an Design System bug, if yes, how do I report it?
+If you see an issue, open a JIRA ticket, otherwise here is our troubleshooting guide.
+
+Do I need to do something special to get components to work in mobile?
+One of our principles is to create components that are responsive. We do not have desktop components and mobile components - we have components that are optimized everywhere. So feel free to use our components for mobile!
+
+Where do I go if I've read everything and still have an issue/question?
+BH Gurus are detail oriented, problem solving, cape wearing UX advocates ready to help with any questions. Whether rain or shine, HipChat or email, they are here to guide you on your journey.
```

Comment:
- [ ] sub hipchat for Google Chat
- [ ] provide link to Google Chat room https://chat.google.com/room/AAAAVTQxM7Y